pom.xml

```diff
@@ -84,8 +81,7 @@
         <dependency>
             <groupId>net.milkbowl.vault</groupId>
             <artifactId>Vault</artifactId>
-            <version>1.4.2-SNAPSHOT</version>
-            <scope>provided</scope>
+            <version>1.5.4</version>
```

Comment:
Scope is related to how it's stored /after/ downloading, it has nothing to do with whether it downloads properly.